Nodejs/Product/Nodejs/NodejsConstants.cs

```diff
@@ -102,9 +100,9 @@ internal static class NodeProjectProperty
         public const string ScriptArguments = "ScriptArguments";
         public const string StartWebBrowser = "StartWebBrowser";
         public const string TypeScriptCfgProperty = "CfgPropertyPagesGuidsAddTypeScript";
-        public const string TypeScriptModuleKind = "TypeScriptModuleKind";
-        public const string TypeScriptOutDir = "TypeScriptOutDir";
-        public const string TypeScriptSourceMap = "TypeScriptSourceMap";
+        // This property is used by the SDK and should be removed. Nevertheless, this is also used by the TestAdapter to find the JS tests. 
+        // My preference is to get the ts configuration to find out the location but might be easier to just repurpose the property.
+        public const string TypeScriptOutDir = "TypeScriptOutDir"; 
```

Comment:
This essentially means that the user will need to specify where the outdir is located for tests files on the njsproj file instead of using the tsconfig file.

Reply:
This has always been true, right?

---
In reply to: [246554781](https://github.com/Microsoft/nodejstools/pull/2123#discussion_r246554781) [](ancestors = 246554781)

Reply:
It has.